src/app/settings.tsx

```diff
@@ -1,11 +1,20 @@
-import { Text } from 'react-native';
+import { Redirect } from 'expo-router';
+import { Text, View } from 'react-native';
+import { Button } from 'react-native-elements';
 import { SafeAreaView } from 'react-native-safe-area-context';
+import { useSession } from '../utils/AuthContext';
 import globalStyles from '../styles/globalStyles';
 
 function SettingsScreen() {
+  const { session, signOut } = useSession();
+
+  if (!session) return <Redirect href="/auth/login" />;
   return (
     <SafeAreaView style={globalStyles.container}>
       <Text style={globalStyles.h1}>Settings</Text>
+      <View>
+        <Button title="Sign Out" onPress={signOut} />
+      </View>
     </SafeAreaView>
   );
```

Comment:
Can you update the settings screen so that users are able to update their profile details there in addition to signing out? It should be almost exactly the same as the onboarding screen.